qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -60,6 +60,15 @@
                                 cpu_model_flags += ",intel_pt=on"
                             i440fx:
                                 cpu_model_flags += ",intel_pt=on,min-level=0x14"
+                - test_SPR_flags:
+                    required_qemu = [5.2.0, )
+                    flags = "serialize avx512_vp2intersect avx512_fp16 tsxldtr"
```

Comment:
Would you pls double check 'tsxldtr'? I don't see you test this flag in the rfe bug.

Reply:
> Would you pls double check 'tsxldtr'? I don't see you test this flag in the rfe bug.

As we discuss in bz, the 'tsxdtr' will can be checked on host(need to be upgrade) and I can check it in code now.